From 5166ba3bde3099e8592db85045650e7af8631e4d Mon Sep 17 00:00:00 2001
From: Doronin Stanislav <mugisbrows@gmail.com>
Date: Wed, 11 Jan 2023 16:23:10 +0300
Subject: [PATCH] lto linker option

---
 mkspecs/win32-g++/qmake.conf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mkspecs/win32-g++/qmake.conf b/mkspecs/win32-g++/qmake.conf
index 6909b3b..cddbc6c 100644
--- a/mkspecs/win32-g++/qmake.conf
+++ b/mkspecs/win32-g++/qmake.conf
@@ -27,7 +27,7 @@ QMAKE_LEX		= flex
 QMAKE_LEXFLAGS		=
 QMAKE_YACC		= byacc
 QMAKE_YACCFLAGS		= -d
-QMAKE_CFLAGS		= -pipe
+QMAKE_CFLAGS		= -pipe -flto
 QMAKE_CFLAGS_DEPS	= -M
 QMAKE_CFLAGS_WARN_ON	= -Wall -Wextra
 QMAKE_CFLAGS_WARN_OFF	= -w
@@ -60,7 +60,7 @@ QMAKE_RUN_CXX_IMP	= $(CXX) -c $(CXXFLAGS) $(INCPATH) -o $@ $<
 
 QMAKE_LINK		= $${CROSS_COMPILE}g++
 QMAKE_LINK_C		= $${CROSS_COMPILE}gcc
-QMAKE_LFLAGS		=
+QMAKE_LFLAGS		= -flto
 QMAKE_LFLAGS_EXCEPTIONS_ON = -mthreads
 QMAKE_LFLAGS_EXCEPTIONS_OFF =
 QMAKE_LFLAGS_RELEASE	= -Wl,-s
-- 
2.32.0.windows.2

