From 87be130db4e3903501c7e92abb1396002870753f Mon Sep 17 00:00:00 2001
From: Doronin Stanislav <mugisbrows@gmail.com>
Date: Wed, 11 Mar 2026 17:18:24 +0300
Subject: [PATCH] pg flag in specs

---
 mkspecs/win32-g++/qmake.conf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mkspecs/win32-g++/qmake.conf b/mkspecs/win32-g++/qmake.conf
index 6909b3b..3923974 100644
--- a/mkspecs/win32-g++/qmake.conf
+++ b/mkspecs/win32-g++/qmake.conf
@@ -36,7 +36,7 @@ QMAKE_CFLAGS_DEBUG	= -g
 QMAKE_CFLAGS_YACC	= -Wno-unused -Wno-parentheses
 
 QMAKE_CXX		= $${CROSS_COMPILE}g++
-QMAKE_CXXFLAGS		= $$QMAKE_CFLAGS
+QMAKE_CXXFLAGS		= $$QMAKE_CFLAGS -pg
 QMAKE_CXXFLAGS_DEPS	= $$QMAKE_CFLAGS_DEPS
 QMAKE_CXXFLAGS_WARN_ON	= $$QMAKE_CFLAGS_WARN_ON
 QMAKE_CXXFLAGS_WARN_OFF	= $$QMAKE_CFLAGS_WARN_OFF
@@ -60,7 +60,7 @@ QMAKE_RUN_CXX_IMP	= $(CXX) -c $(CXXFLAGS) $(INCPATH) -o $@ $<
 
 QMAKE_LINK		= $${CROSS_COMPILE}g++
 QMAKE_LINK_C		= $${CROSS_COMPILE}gcc
-QMAKE_LFLAGS		=
+QMAKE_LFLAGS		= -pg
 QMAKE_LFLAGS_EXCEPTIONS_ON = -mthreads
 QMAKE_LFLAGS_EXCEPTIONS_OFF =
 QMAKE_LFLAGS_RELEASE	= -Wl,-s
-- 
2.45.2.windows.1

