From d111e1e93a64375952cb3f92d83727e1747d9315 Mon Sep 17 00:00:00 2001
From: Doronin Stanislav <mugisbrows@gmail.com>
Date: Tue, 10 Jan 2023 23:38:06 +0300
Subject: [PATCH] include winerror.h

---
 src/network/kernel/qnetworkinterface_win.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/network/kernel/qnetworkinterface_win.cpp b/src/network/kernel/qnetworkinterface_win.cpp
index ffbfabf..bfde0f7 100644
--- a/src/network/kernel/qnetworkinterface_win.cpp
+++ b/src/network/kernel/qnetworkinterface_win.cpp
@@ -49,6 +49,7 @@
 #include <qhash.h>
 #include <qurl.h>
 #include <private/qsystemlibrary_p.h>
+#include <winerror.h>
 
 QT_BEGIN_NAMESPACE
 
-- 
2.32.0.windows.2

